app-dicts/myspell-tr/metadata.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<!-- maintainer-needed -->
+	<stabilize-allarches/>
+</pkgmetadata>
```

Comment:
Also just so you know, this is against our QA policy, but I'd imagine it's fine in this case.
https://projects.gentoo.org/qa/policy-guide/maintainer.html#pg0602

Reply:
If that's a problem please add myself as maintainer